
joint_velocity_controllers:
  # 控制左前轮
  left_front_wheel_velocity_controller:
    type: "effort_controllers/JointVelocityController"
    joint: "left_front_wheel_joint"
    pid:
      p: 100.0
      i: 0.5
      d: 0.0
      i_max: 10.0
      i_min: -10.0
      antiwindup: true

  # 控制右前轮
  right_front_wheel_velocity_controller:
    type: "effort_controllers/JointVelocityController"
    joint: "right_front_wheel_joint"
    pid:
      p: 100.0
      i: 0.5
      d: 0.0
      i_max: 10.0
      i_min: -10.0
      antiwindup: true

  # 控制左后轮
  left_back_wheel_velocity_controller:
    type: "effort_controllers/JointVelocityController"
    joint: "left_back_wheel_joint"
    pid:
      p: 100.0
      i: 0.5
      d: 0.0
      i_max: 10.0
      i_min: -10.0
      antiwindup: true

  # 控制右后轮
  right_back_wheel_velocity_controller:
    type: "effort_controllers/JointVelocityController"
    joint: "right_back_wheel_joint"
    pid:
      p: 100.0
      i: 0.5
      d: 0.0
      i_max: 10.0
      i_min: -10.0
      antiwindup: true
